.devtools/assemble.sh

```
@@ -0,0 +1,188 @@
+
+info "Validate tools."
+! command -v git >/dev/null && fail "ERROR: Git is required for this script to run." && exit 1
+! command -v php >/dev/null && fail "ERROR: PHP is required for this script to run." && exit 1
+! command -v composer >/dev/null && fail "ERROR: Composer (https://getcomposer.org/) is required for this script to run." && exit 1
+! command -v jq >/dev/null && fail "ERROR: jq (https://stedolan.github.io/jq/) is required for this script to run." && exit 1
+pass "Tools are valid."
+
+# Set the sed options based on the OS.
+sed_opts=(-i) && [ "$(uname)" == "Darwin" ] && sed_opts=(-i '')
+
+extension="civictheme_sdc"
```

Shellcheck (0.10.0)

[warning] extension appears unused. Verify use (or export if used externally).

(SC2034)